layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en"><head>
+  <meta charset="utf-8">
+  <title>#BlackLivesMatter</title>
+
+  <link rel="dns-prefetch" href="https://fonts.googleapis.com">
+  <link rel="dns-prefetch" href="https://fonts.gstatic.com">
+  <link rel="dns-prefetch" href="https://api.github.com">
+
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600&amp;display=fallback">
+  <link rel="stylesheet" href="https://nodejs.org/static/css/styles.css">
+
+  <meta name="author" content="Node.js">
+  <meta name="robots" content="index, follow">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="#BlackLivesMatter">
+
+  <link rel="apple-touch-icon" href="https://nodejs.org/static/images/favicons/apple-touch-icon.png" sizes="180x180">
+  <link rel="icon" href="https://nodejs.org/static/images/favicons/favicon-32x32.png" sizes="32x32" type="image/png">
+  <link rel="icon" href="https://nodejs.org/static/images/favicons/favicon-16x16.png" sizes="16x16" type="image/png">
+  <link rel="manifest" href="https://nodejs.org/static/images/favicons/manifest.json">
+  <link rel="mask-icon" href="https://nodejs.org/static/images/favicons/safari-pinned-tab.svg" color="#43853d">
+  <link rel="shortcut icon" href="https://nodejs.org/static/images/favicons/favicon.ico">
+  <meta name="msapplication-TileColor" content="#333333">
+  <meta name="msapplication-config" content="https://nodejs.org/static/images/favicons/browserconfig.xml">
+  <meta name="theme-color" content="#333333">
+
+  <meta property="og:site_name" content="Node.js">
+  <meta property="og:title" content="#BlackLivesMatter">
+  <meta property="og:image" content="https://nodejs.org/static/images/logo-hexagon-card.png">
+  <meta property="og:image:type" content="image/png">
+  <meta property="og:image:width" content="224">
+  <meta property="og:image:height" content="256">
+  <meta property="og:description" content="#BlackLivesMatter">
+
+  <meta name="twitter:card" content="summary">
+  <meta name="twitter:site" content="@nodejs">
+  <meta name="twitter:title" content="#BlackLivesMatter">
+  <meta name="twitter:image" content="https://nodejs.org/static/images/logo-hexagon-card.png">
+  <meta name="twitter:image:alt" content="The Node.js Hexagon Logo">
+  <style>
+    body {
+      background: #333;
+      color: #fff;
+      min-width: 80%;
+      max-width: 50%;
+      margin: 0 auto;
+    }
+
+    section {
+      padding-bottom: 30px;
+    }
+
+    ul.humans > li {
+      list-style: none;
+      font-size: 1.7rem
+    }
+
+    h1 {
+      font-size: 5rem
+    }
+
+    .center {
+      margin: 0 auto
+    }
+
+    .space {
+      padding-top: 4rem;
+      padding-bottom: 2rem;
+    }
+
+    footer {
+      margin-bottom: 2rem;
+    }
+  </style>
+</head>
+
+<body>
+  <h1 class="center space">#BlackLivesMatter</h1>
+  <section id="lives" class='center'>
+    <ul class="humans">
+      <li>Ahmaud Arbery</li>
```

Comment:
Is the goal here to list only victims of police brutality in the US, or in other countries as well? This is not a US-only problem, but the discourse and momentum (at least in the US) right now is US-centered (I'm not sure how the discourse is going on other countries).

I don't have an answer to this and don't think it should block the PR, but I was wondering if others thought about it.

Reply:
@mmarchini My feedback is that we should welcome additions to the list with open arms.

Reply:
I think we should probably focus on the problem that is currently at the forefront, *but* I also do not believe this needs to be something we look at with scarcity. Templates are effectively free to us - they costs us very little to build and ship. We can look at this and do it more with precise and targeted support in the future in addition to in this instance.

Reply:
That's mostly aligned with what I'm thinking, although I'm still reflecting on the question above. I don't think discussing it further here would be productive though, and I don't want to block this PR. Besides, we can make changes later if we believe those are necessary.

Reply:
My concern here is that we can bikeshed on this continually without ever posting anything. We can iterate on the list if necessary, but let's please get *something* published.

Reply:
FWIW @MylesBorins I've added a link to https://ebwiki.org

Reply:
I have a few suggestions, which you can take or leave. I think that @mmarchini 's point is very important, and should be addressed.

I think it should be addressed with a note as opposed to trying to get every single person in the world listed.

```
I would recommend something like this:

Note: We've listed the names of the Black folks who have been murdered by police brutality in the [U.S./North American] since [X], both because it is the origin of the Black Lives Matter movement and currently the heart of the resistance.

With that said, we fully recognise that white supremacy and police brutality is a global problem. While we have not listed all of Black people who have lost their lives to police murder globally here, their lives mattered. For a more global extensive list, please visit [url](url).

Reply:
Also WRT to gender breakdown, this isn't a panel. Accuracy in the frame you set is most important (so by date, then by location). It's VERY important here to not partake in the erasure, particularly because the people who typically get erased experience multiple marginalisations like of gender and orientation.

Reply:
Working on a suggestion for the above now. Just validating the year of the earliest murder before I add it.